synthetic_data/distinct_generators/float_generator.py

```diff
@@ -10,7 +11,7 @@ def random_floats(
     num_rows: int = 1,
 ) -> np.array:
     """
-    Randomly generates an array of floats between a min and max value
+    Randomly generates an array of floats between a min and max value.
```

Comment:
nit pick but I might think about saying `the min and max value` because it is generating between *the* min / max value provided by the user